client/landing/stepper/declarative-flow/helpers/index.ts

```diff
@@ -3,11 +3,13 @@ import { STEPS } from '../internals/steps';
 export const shouldRedirectToSiteMigration = (
 	step: string,
 	platform: string,
+	locale: string,
 	origin?: string | null
 ) => {
 	return (
 		step === STEPS.IMPORT_LIST.slug &&
 		platform === 'wordpress' &&
+		locale === 'en' &&
```

Comment:
@Automattic/serenity, can we look into making this check whether we have any English locale, not just `en`?

Reply:
This request was tackled on https://github.com/Automattic/wp-calypso/pull/89035